assets/js/components/OverlayNotification/AnalyticsAndAdSenseAccountsDetectedAsLinkedOverlayNotification.js

```diff
@@ -203,34 +196,15 @@ export default function AnalyticsAndAdSenseAccountsDetectedAsLinkedOverlayNotifi
 		}, 50 );
 	};
 
-	useEffect( () => {
-		if ( shouldShowNotification && ! isShowingNotification ) {
-			// If the conditions to show this notification are met AND no other
-			// notifications are showing, show this notification.
-			setOverlayNotificationToShow(
-				ANALYTICS_ADSENSE_LINKED_OVERLAY_NOTIFICATION
-			);
-
-			trackEvent(
-				`${ viewContext }_top-earning-pages-widget`,
-				'view_overlay_CTA'
-			);
-		}
-	}, [
-		shouldShowNotification,
-		isShowingNotification,
-		setOverlayNotificationToShow,
-		viewContext,
-	] );
-
-	if ( ! shouldShowNotification || ! isShowingNotification ) {
-		return null;
-	}
-
 	return (
-		<OverlayNotification animateNotification={ isShowingNotification }>
-			<AnalyticsAdsenseLinkedGraphic />
-
+		<OverlayNotification
+			shouldShowNotification={ shouldShowNotification }
+			GraphicDesktop={ AnalyticsAdsenseLinkedGraphicDesktop }
+			GraphicMobile={ AnalyticsAdsenseLinkedGraphicMobile }
+			notificationID={ ANALYTICS_ADSENSE_LINKED_OVERLAY_NOTIFICATION }
+			gaEventCategory={ `${ viewContext }_top-earning-pages-widget` }
+			gaEventAction="view_overlay_CTA"
```

Comment:
I'm not happy with the GA event specific props here, but since the `isShowingNotification` logic for mobile and desktop are slightly different, we will have to either use props here, or duplicate the detection logic for mobile and desktop using the breakpoints here.

Reply:
That makes more sense, I'll get it updated.

Reply:
Looks great now, thanks!